### ISSUE TITLE
All graphs have only a node type

### ISSUE THREAD

**Issue by LucaCappelletti94** [Knowledge-Graph-Hub/kg-obo#111]
All of the 149 graphs currently available only have a single node type, [`'biolink:NamedThing'`](https://biolink.github.io/biolink-model/docs/NamedThing.html). This is most likely a transformation bug.

**Comment by caufieldjh** [comment]
You're right - this is a result of the kgx transformation. Resolving arbitrary ontology classes to their respective Biolink types isn't currently possible, as far as I know, but it would be great to have more specific nice types where we can.

See also https://github.com/biolink/kgx/issues/305

**Comment by caufieldjh** [comment]
I think there should be *some* other node types though, since there are some OBOs where the biolink semantic types have been clearly mapped. I'm guessing that the nodes/edges for HP are still all NamedThing, though that should certainly contain some other types.
